mise.toml

```diff
@@ -18,7 +18,7 @@ just = "1.37.0"
 "go:github.com/ethereum/go-ethereum/cmd/abigen" = "1.10.25"
 "go:gotest.tools/gotestsum" = "1.12.0"
 "go:github.com/vektra/mockery/v2" = "2.46.0"
-"go:github.com/golangci/golangci-lint/cmd/golangci-lint" = "1.64.5"
+"go:github.com/golangci/golangci-lint/cmd/golangci-lint" = "1.61.0"
```

Comment:
why this?

Reply:
I updated this during op-geth upstream merge work previously, but I don't think it was necessary and seems to conflict with our choice to not update the go version yet.